crates/server/src/services/core.rs

```diff
@@ -248,12 +248,15 @@ impl CoreService {
 
         store.validate_operator_record(&log_id, &record_id).await?;
 
-        let leaf = LogLeaf { log_id, record_id };
+        let leaf = LogLeaf {
+            log_id: log_id.clone(),
```

Comment:
I think this clone is unnecessary (i.e. we can back out this change along with the change to inserting into the map below).